### PR TITLE
AP_Motors: reuse spool_step for code refactoring

### PR DESCRIPTION
**Pull request by chobitsfan**
I think it may improve code readability
tested on CubeBlack
[2022-07-08 10-28-02.zip](https://github.com/ArduPilot/ardupilot/files/9068482/2022-07-08.10-28-02.zip)

### CODE REVIEW

**Review by rmackay9 — APPROVED** [review]

**Comment by rmackay9** [comment]
Looks good to me thanks!

File: libraries/AP_Motors/AP_MotorsMulticopter.cpp [hunk 1]
```diff
@@ -556,6 +556,7 @@ void AP_MotorsMulticopter::output_logic()
         _spool_up_time.set(0.05);
     }
 
+    float spool_step = 1.0f / (_spool_up_time * _loop_rate);
```
Comment (IamPete1) [inline libraries/AP_Motors/AP_MotorsMulticopter.cpp:559]:
```suggestion
    const float spool_up_step = 1.0f / (_spool_up_time * _loop_rate);
```

**Review by IamPete1 — APPROVED** [review]
Change looks good, you could make it const if you like.

File: libraries/AP_Motors/AP_MotorsMulticopter.cpp (hunk 1 above)
Comment (chobitsfan) [reply libraries/AP_Motors/AP_MotorsMulticopter.cpp]: Yes, it is better with const. Thank you

**Review by Gone4Dirt — CHANGES_REQUESTED** [review]

File: libraries/AP_Motors/AP_MotorsMulticopter.cpp [hunk 2]
```diff
@@ -556,6 +556,7 @@ void AP_MotorsMulticopter::output_logic()
         _spool_up_time.set(0.05);
     }
 
+    const float spool_up_step = 1.0f / (_spool_up_time * _loop_rate);
```
[the remark this answers is not in the document]
Comment (chobitsfan) [reply libraries/AP_Motors/AP_MotorsMulticopter.cpp]: Thank you. it is a typo

**Comment by rmackay9** [comment]
Merged, thanks!

**Pull request merged**